### PR TITLE
Refactor federated attestation signing logic

### LINKED ISSUE #9929: Move the attestation signing logic into the identity sdk
Move helper functions in `fed-attestation-utils` into the identity sdk

### PR DESCRIPTION
**Pull request by Jesse-Sawa**
closes [#9929](https://github.com/celo-org/celo-monorepo/issues/9929)

### CODE REVIEW

File: packages/sdk/identity/src/federatedAttestations/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import * as utils from './signatureUtilities'
+
+export const FederatedAttestations = {
+  utils,
+}
```
Comment (Jesse-Sawa) [inline packages/sdk/identity/src/federatedAttestations/index.ts]: Related convenience functions will be added here

**Review by isabellewei — CHANGES_REQUESTED** [review]
I think you need to add `@celo/identity` to the `protocol` `package.json`, as it currently isn't being imported

**Comment by Jesse-Sawa** [comment]
> I think you need to add `@celo/identity` to the `protocol` `package.json`, as it currently isn't being imported

refactored this PR so /protocol doesn't pull in the identity SDK

File: packages/sdk/utils/src/typed-data-constructors.ts [hunk 2]
```diff
@@ -59,3 +60,43 @@ export const authorizeSigner = ({
...
+) => {
+  return {
+    types: {
+      EIP712Domain: [
+        { name: 'name', type: 'string' },
+        { name: 'version', type: 'string' },
+        { name: 'chainId', type: 'uint256' },
+        { name: 'verifyingContract', type: 'address' },
+      ],
+      OwnershipAttestation: [
```
Comment (alecps) [inline packages/sdk/utils/src/typed-data-constructors.ts:85]: Is there a reason we don't call this AttestationDetails or vice versa?

**Review by alecps — APPROVED** [review]
Looks great!

File: packages/sdk/utils/src/typed-data-constructors.ts (hunk 2 above)
Comment (isabellewei) [reply packages/sdk/utils/src/typed-data-constructors.ts:85]: needs to match the variable name declared in the FA smart contract

**Review by isabellewei — APPROVED** [review]
🎉

**Pull request merged**